docs/index.md

```diff
@@ -83,9 +83,21 @@ Try on github {fab}`github;fa-xl`
 A youtube video by the kedro team to introduce the plugin, with live coding.
 
 +++
-See on youtube {fab}`youtube;fa-xl`
+Watch on youtube {fab}`youtube;fa-xl`
 :::
 
+:::{grid-item-card}
+:link: https://youtu.be/mIfJR3CdBUE
+:link-type: url
+:class-header: bg-light
+
+{fas}`fa-solid fa-video fa-xl;pst-color-primary` Tackling the ML Reproducibility Curse
+^^^
+
+A community video showcasing the power of the Kedro-MLflow plugin.
```

Comment:
As discussed I'd be completly fine with something shamelessly naming you, e.g. : 

```suggestion
A community video by [Oleg Litvinov](https://github.com/OlegBEZb) showcasing how to use the Kedro-MLflow plugin on an end to end project.
```

but it's up to you!